api/src/libs/githubInvitationInit.ts

```diff
@@ -0,0 +1,162 @@
+import { createAppAuth } from '@octokit/auth'
+import { request } from '@octokit/request'
+import { getGithubPrivateKey } from '../config/githubConfig'
+import { logger } from '@bcgov/common-nodejs-utils';
+
+// cached value
+type APPInitailValue = {
```

Comment:
Typo 
```suggestion
type APPInitialValue = {
```
Also, should APP be App?
```suggestion
type AppInitialValue = {
```